### PR TITLE
Check if /usr/bin symlink is present to fix instance reuse issue on older Pocketbooks

### LINKED ISSUE #11760: Can't open books from library when files are associated on some Pocketbooks
* KOReader version: 2024.04
* Device: Pocketbook Basic Touch

#### Issue
I'm not sure why, but when I set up file associations (Device -> Associate file extensions -> epub, epub3, pdf) so that KOreader would open automatically when starting the reader, I can no longer reliably open books from the system library application or from the "recent books" on home screen. Sometimes it works, but most of the time nothing happens (the given book starts opening only when I manually start KOreader). I should also note that opening books on reader startup works flawlessly, and I have no such issues whatsoever on my other Pocketbook reader (Aqua 2).

If you have any ideas what might help with diagnostics, I'd be most happy to try it out.

### PR DESCRIPTION
**Pull request by sdasda7777**
I believe this fixes #11760

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/11764)

### CODE REVIEW

**Comment by Frenzie** [comment]
/usr/bin sounds theoretically preferable to me (with a fallback for older devices) but I'll refrain from opining on PB.

**Comment by sdasda7777** [comment]
Makes sense to me, I'll look into it.

File: platform/pocketbook/koreader.app [hunk 1]
```diff
@@ -15,7 +15,7 @@ export KO_PATH_OPEN_BOOK="/tmp/.koreader.open"
 INSTANCE_PID=$(cat /tmp/koreader.pid 2>/dev/null)
 if [ "${INSTANCE_PID}" != "" ] && [ -e "/proc/${INSTANCE_PID}" ]; then
     echo "$@" >"${KO_PATH_OPEN_BOOK}"
-    exec /ebrmain/bin/iv2sh SetActiveTask "${INSTANCE_PID}" 0
+    exec $(if [ -f "/usr/bin/iv2sh" ]; then echo /usr/bin/iv2sh; else /ebrmain/bin/iv2sh; fi) SetActiveTask "${INSTANCE_PID}" 0
```
Comment (benoit-pierre) [inline platform/pocketbook/koreader.app]: Missing an echo in the else branch?

Comment (sdasda7777) [reply platform/pocketbook/koreader.app]: Indeed, already fixed

Comment (benoit-pierre) [reply platform/pocketbook/koreader.app]: Yep, and I had already resolved the conversation. ;)

**Comment by sdasda7777** [comment]
Hi, does anyone have any concerns as to why this should not be merged?

**Comment by Frenzie** [comment]
No concerns, just waiting for potential comments. Thanks!

**Pull request merged**